com.shadps4emu.ShadPS4.yaml

```
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+id: com.shadps4emu.ShadPS4
```

We have a website: https://shadps4.net/.
Plus, should we make shadPS4 name more consistent?
```suggestion
id: net.shadps4.shadPS4
```

---

Manifest file name too:
```
com.shadps4emu.ShadPS4.yaml
```
->
```
net.shadps4.shadPS4.yaml
```